core/src/main/resources/jenkins/views/JenkinsHeader/headerContent.jelly

```diff
@@ -0,0 +1,80 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:i="jelly:fmt" xmlns:x="jelly:xml">
+	<header id="header" class="page-header">
```

Comment:
don't mix tabs and spaces

Reply:
Fixed in https://github.com/jenkinsci/jenkins/pull/5909/commits/284abf7c8fdcc2da6e16e72e8c9a9ec33895aaab